includes/class-wc-payment-token-swedbank-pay.php

```diff
@@ -168,9 +172,9 @@ public static function wc_get_account_saved_payment_methods_list_item( $item, $p
 	 * @return void
 	 */
 	public static function wc_account_payment_methods_column_method( $method ) {
-		if ( $method['method']['gateway'] === 'payex_psp_cc' ) {
+		if ( 'payex_psp_cc' === $method['method']['gateway'] ) {
 			$token = new WC_Payment_Token_Swedbank_Pay( $method['method']['id'] );
-			echo $token->get_display_name();
+			echo esc_html( $token->get_display_name() );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)